lib/takeaway.rb

```diff
@@ -0,0 +1,40 @@
+# require_relative 'send_sms'
+
```

Comment:
I think adding some comments and a README with irb instructions would be great for anyone who doesn't understand your code :)